### PR TITLE
Reword index to make adding methods slightly easier

### PR DESCRIPTION
**Pull request by klamping**
Small little change that should, in theory, make adding new assertions a little easier.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ import immediately from './chains/immediately';
...
 export default function (client) {
     return function chaiWebdriverIO(chai, utils) {
-        there(client, chai, utils);
-        visible(client, chai, utils);
-        count(client, chai, utils);
-        text(client, chai, utils);
-        immediately(client, chai, utils);
+        let methodsToAdd = [there, visible, count, text, immediately];
+
+        methodsToAdd.map(function (methodToAdd) {
```
Comment (marcodejongh) [inline src/index.js]: Could you change this to ```forEach``? ```map``` should be used when mapping an array to a new array but that's not what this is doing

**Review by marcodejongh — CHANGES_REQUESTED** [review]

**Comment by klamping** [comment]
That's a very good point. Updated.

**Review by marcodejongh — APPROVED** [review]

**Comment by marcodejongh** [comment]
@klamping Thanks, because there's no change in functionality I'll defer releasing it till later

**Comment by klamping** [comment]
@marcodejongh sounds good. I'll *hopefully* have some real updates soon.

**Pull request merged**